packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Policies
+meta:
+  - name: description
+    content: An overview of what policies are and how you can use them.
+---
+## What are policies
+
+Policies help you manage access to your applications and APIs. You can restrict access based on a number of conditions such as user and group membership, device, location, or time. You can also require more authentication steps for access to sensitive applications, such as confirmation of a push notification to a mobile device or re-authentication through an SMS one-time passcode.
+
+Policies are objects that Okta evaluates when a request is made. Rules in the policies define permissions that determine whether the request is allowed or denied. Policies are stored in Okta as JSON documents.
+
+There are many possibilities for policy use:
+
+* Create authorization rules based on complex logic using conditions
+* Determine extra levels of authentication that must be performed before a specific Okta application can be accessed
+* Maintain a white-list of users and deny access based on multiple conditions
+* Change the returned scopes of the access token and add claims to it and to the ID token using [inline hooks](/docs/concepts/inline-hooks/)
+* Notify other systems through an API when a sign in happens in real-time
+* Query for user information from a database/API using scopes and add it to the user profile object
+
+Okta supports six types of policies:
+
+* [Okta Sign On Policy](/docs/reference/api/policy/#okta-sign-on-policy) &mdash; Controls who can sign in and how a user is allowed to sign in to Okta, including whether they are challenged for MFA and how long they are allowed to remain signed in before re-authenticating.
+
+* [App Sign On Policy](https://help.okta.com/en/prod/okta_help_CSH.htm#App_Based_Signon) &mdash; Determines the extra levels of authentication that you may want performed before an application can be accessed.
+
+* [Okta MFA Policy](/docs/reference/api/policy/#multifactor-mfa-enrollment-policy) &mdash; Controls which MFA methods are available for a user, as well as when a user may enroll in a particular factor.
+
+* [Password Policy](/docs/reference/api/policy/#password-policy) &mdash; Determines the requirements for a user's password length and complexity, as well as the frequency with which a password must be changed. This policy also governs the recovery operations that may be performed by the user, including change password, reset (forgot) password, and self-service password unlock.
+
+* [IdP Discovery Policy](/docs/reference/api/policy/#idp-discovery-policy) &mdash; Determines where to route users when they are attempting to sign in to your org. Users can be routed to a variety of identity providers (SAML2, IWA, AgentlessDSSO, X509, APPLE, FACEBOOK, GOOGLE, LINKEDIN, MICROSOFT, OIDC).
```

Comment:
I would also add a note here saying that this policy is _not_ for doing authentication or authorization. Only for determining where a user should be routed. You can not control access with an IdP Discovery Policy.